dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/infer/DescriptorLanguageInferrer.java

```diff
@@ -151,8 +151,12 @@ protected boolean isReferencedPath(String possiblePath) {
     }
 
     private Optional<Path> toAbsolutePath(Path currentPath, String relativeOrAbsolutePath) {
+        Path parentPath = currentPath.getParent();
```

Comment:
A little puzzled, this seems to never resolve to the previous code no matter what condition. Was this completely broken and just unnoticed?

Reply:
Maybe it's the same if the `currentPath` is the root?